jerry-core/ecma/base/ecma-lcache.c

```diff
@@ -54,11 +65,11 @@ ecma_lcache_invalidate_entry (ecma_lcache_hash_entry_t *entry_p) /**< entry to i
  */
 static inline size_t JERRY_ATTR_ALWAYS_INLINE
 ecma_lcache_row_index (jmem_cpointer_t object_cp, /**< compressed pointer to object */
-                       lit_string_hash_t name_hash) /**< property name hash */
+                       jmem_cpointer_t name_cp) /**< property name hash */
```

Comment:
It looks like this is no longer a hash, please update the comment as well.

Reply:
Good catch, thanks!